Tests/MapboxNavigationTests/VanishingRouteLineTests.swift

```diff
@@ -0,0 +1,262 @@
+import XCTest
+import MapboxDirections
+import TestHelper
+import Turf
+import MapboxMaps
+@testable import MapboxNavigation
+@testable import MapboxCoreNavigation
```

Comment:
There is no need to directly use Turf:
```suggestion
import XCTest
import MapboxDirections
import TestHelper
import MapboxMaps
@testable import MapboxNavigation
@testable import MapboxCoreNavigation
```

Reply:
Thanks for the suggestion, the extra directly importing is removed.